posix-configs/bebop/px4.config

```diff
@@ -36,4 +40,4 @@ mavlink stream -u 14556 -s HIGHRES_IMU -r 50
 mavlink stream -u 14556 -s ATTITUDE -r 50
 df_bebop_bus_wrapper start
 mavlink boot_complete
-sdlog2 start -e -b 20 -t
+logger start -b 20 -e -t
```

Comment:
@LorenzMeier Sure we can, for once we are not out or RAM ;-) I've checked from dropouts and haven't seen any (logger status) but we can still increase it a bit.